### ISSUE TITLE
jieba question

### ISSUE THREAD

**Issue by kas84** [Siddhant-K-code/OpenAI-bring-your-own-data#1]
Why are you using jieba here? That shouldn't be needed unless you're using Chinese text, right?

**Comment by Siddhant-K-code** [comment]
Yeah, I still need to do some cleanups & optimizations. But, this was purely just to parse through some specific research papers/txt
